app/tests/test_questions.py

```diff
@@ -0,0 +1,43 @@
+from fastapi.testclient import TestClient
+import unittest
+import json
+from mongoengine import connect, disconnect
+from main import app
+
+client = TestClient(app)
+
+# We are currently not providing an endpoint for creating questions and the only way to
+# create a question is through the quiz endpoint which is why we are using the quiz endpoint
+# to create dummy questions
```

Comment:
move this to where you are actually making the API call no? How will someone understand the context here?